modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderImport.testcase

```diff
@@ -133,14 +133,74 @@ definition {
 		}
 	}
 
-	@description = "[Please Update]"
-	@ignore = "Test Stub"
+	@description = "Create multiple orders and use the search bar to verify if it works correctly"
 	@priority = "4"
 	test ImportFromOrderUseSearchBar {
 		property portal.acceptance = "false";
 
-		// TODO COMMERCE-7686 ImportFromOrderUseSearchBar pending implementation
+		task ("Given a Minium site with a buyer user") {
+			User.logoutPG();
+
+			CommerceLogin.miniumLogin(
+				password = "test",
+				urlAppend = "web/minium",
+				userEmailAddress = "buyer@liferay.com");
+		}
+
+		task ("And given 3 completed orders") {
+			for (var product : list "U-Joint:24,Mount:3,Torque Converters:34") {
+				var productName = StringUtil.extractFirst("${product}", ":");
+				var productUnitPrice = StringUtil.extractLast("${product}", ":");
+
+				FrontStore.addInitializerProductToCart(
+					productName = "${productName}",
+					productQuantity = "1");
+
+				CommerceNavigator.gotoMiniCart();
+
+				Button.click(button = "View Details");
+
+				Button.click(button = "Checkout");
+
+				CommerceCheckout.initializersCheckout(
+					newAddressAddress = "Test Address",
+					newAddressCity = "Test City",
+					newAddressCountry = "United States",
+					newAddressName = "Address Name",
+					newAddressZipCode = "Test Zip Code",
+					productName = "${productName}",
+					productQuantity = "1",
+					productUnitPrice = "${productUnitPrice}",
+					shippingMethod = "Standard");
+
+				CommerceNavigator.gotoMiniumSidebarItem(menuItem = "Catalog");
+			}
+		}
+
+		task ("When the buyer tries to import an order") {
+			CommerceNavigator.gotoMiniumSidebarItem(menuItem = "Catalog");
+
+			CommerceAcceleratorsInitializer.createNewOrderFromPendingOrders();
+
+			Click(locator1 = "CommerceEntry#DROPDOWN");
+
+			MenuItem.click(menuItem = "Import from Orders");
+		}
+		
+
+		task ("Then search for the ID of each order and the result should match the order") {
+			IFrame.selectModalFrame();
+
+			var orderIds = CommerceJSONOrdersAPI._getCommerceOrderIds(orderCount = "3");
 
+			for (var orderId : list "${orderIds}") {
+				Search.searchCP(searchTerm = "${orderId}");
+
+				AssertElementPresent(
+					key_entryName = "${orderId}",
+					locator1 = "CommerceEntry#TABLE_LIST_TITLE");
+			}
+		}
```

Comment:
This should be two separate tasks.

1. When the buyer searches the ID of a completed order
2. Then only that order should appear

Also. Can you add asserts that the other orders aren't present. You only need to search for one order.